Cargo.toml

```diff
@@ -18,6 +18,7 @@ azure-iot-sdk = { git = "https://github.com/omnect/azure-iot-sdk.git", tag = "0.
 dotenvy = "0.15"
 env_logger = "0.8"
 enum_dispatch = "0.3"
+futures = "0.3.28"
```

Comment:
currently our strategy is to set only major + minor. are you aware of any reasons to don't so?

Reply:
Default behavior of cargo-add. N.B. it is counter intuitive but 0.3.28 means: 0.3.28 and everything above (<= 0.4), see: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio ; but I'll adjust it :).